aemedge/blocks/footer/footer.css

```diff
@@ -1,48 +1,51 @@
 .block.footer {
-  display: flex;
   background: var(--udexCoreSurfaceHigher);
-  padding-top: var(--udexSpacer48);
-  padding-bottom: var(--udexSpacer40);
-}
-
-.block.footer > div {
   display: flex;
-  position: relative;
-  flex-direction: column;
-  width: 100%;
-  padding-inline: var(--udexGridMargins);
-}
-
-.block.footer .contact-container {
-  flex: 1;
-}
-
-.block.footer .link-group-container {
-  flex: 2;
-}
-
-.block.footer .icon-back-to-top {
-  position: static;
-  height: 32px;
-  width: 32px;
-  padding-top: var(--udexSpacer40);
-}
-
-.block.footer a {
-  color: var(--text-color);
-  text-decoration: none;
-}
-
-.block.footer a:hover {
-  color: var(--sapLink_Hover_Color);
-}
-
-footer .block.link-group p {
-  margin: 0;
+  padding-bottom: var(--udexSpacer40);
+  padding-top: var(--udexSpacer48);
```

Comment:
@eduardseifert Why do have changes in contact and footer files for this PR?

Reply:
1. After introducing the `box-sizing: border-box` styles, the icons were not correctly rendered. I corrected this.
2. Took the possibility to nest `footer.css` and `contact.css` (and chore).